pandas/tests/io/json/test_normalize.py

```diff
@@ -463,6 +463,15 @@ def test_nested_flattening_consistent(self):
         # They should be the same.
         tm.assert_frame_equal(df1, df2)
 
+    def test_nonetype_record_path(self, nulls_fixture):
+        # see gh-30148
```

Comment:
can you add an additional example that has some valid data in the result frame, it is not clear whether this should or should not do anything

Reply:
Will do